client/admin.go

```diff
@@ -28,8 +28,8 @@ const (
 type AdminClient interface {
 	AddOrUpdatePolicy(ctx context.Context, policies *PolicySet) error
 	AuditLogs(ctx context.Context, opts AuditLogOptions) (<-chan *AuditLogEntry, error)
-	ListPolicies(ctx context.Context) ([]string, error)
-	GetPolicy(ctx context.Context, ids ...string) ([]*policyv1.Policy, error)
+	ListPolicies(ctx context.Context, includeDisabled bool) ([]string, error)
```

Comment:
This would be a breaking change to existing users. I think we should declare it like `ListPolicies(context.Context, ...ListPoliciesOption)`. 

There would be just one `ListPoliciesOption` and that would be the one to include disabled policies in the output.

Reply:
It would be confusing for the user, wouldn't it? If we are keen on this, at least we could return an error in case more than one `opt` are provided to the function, WDYT?

Reply:
Why do you think it would be confusing?

We don't need to worry about how many options are passed to the function. The last one would always win. That's the expected behaviour for functional options in Go.